examples/deployments/databricks.py

```diff
@@ -1,14 +1,79 @@
+"""
+Usage
+-----
+databricks secrets create-scope <scope>
+databricks secrets put-secret <scope> openai-api-key --string-value $OPENAI_API_KEY
+python examples/deployments/databricks.py --secret <scope>/openai-api-key
+-----
+"""
+import argparse
+import uuid
+
 from mlflow.deployments import get_deploy_client
 
 
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--secret", type=str, help="Secret (e.g. secrets/scope/key)")
+    return parser.parse_args()
+
+
 def main():
+    args = parse_args()
     client = get_deploy_client("databricks")
+    name = f"test-endpoint-{uuid.uuid4()}"
     client.create_endpoint(
-        name="gpt4-chat",
+        name=name,
         config={
-            # TODO: doesn't work yet
+            "served_entities": [
+                {
+                    "name": "test",
+                    "external_model": {
+                        "name": "gpt-4",
+                        "provider": "openai",
+                        "openai_config": {
+                            "openai_api_key": "{{" + args.secret + "}}",
+                        },
+                    },
+                }
+            ],
+            "task": "llm/v1/chat",
+            "tags": [
+                {
+                    "key": "foo",
+                    "value": "bar",
+                }
+            ],
```

Comment:
`create_endopoint` worked yesterday, but doesn't today. Did we update something? @mingyu89 

```
requests.exceptions.HTTPError: 400 Client Error: Bad Request for url: https://e2-dogfood.staging.cloud.databricks.com/api/2.0/serving-endpoints/. Response text: {"error_code":"INVALID_PARAMETER_VALUE","message":"Please provide a valid endpoint task type."}
```

Reply:
or I'm just missing something?

Reply:
There's a new API update got merged yesterday. This notebooks is the latest state:
https://e2-dogfood.staging.cloud.databricks.com/?o=6051921418418893#notebook/4169768892691893/command/4169768892691894

Basically, the "task" is moved from a top level parameter of the request to the external_model object.
And the `openai_api_key` field only takes databricks secrets with format `{{secrets/<scope>/<key>}}`. It no longer takes plain text API token.

Reply:
got it, updating the code

Reply:
Updated. The example is working fine now.